### PR TITLE
WAVE2-937 Added hook_uninstall() to clean-up now unused fields and tables

### PR DESCRIPTION
**Pull request by dpfitzsi**
This will resolve install errors on Live

### CODE REVIEW

**Comment by dpfitzsi** [comment]
Hi Balraj, please take a look at this hook_uninstall() function for the fgs module. It will clean up the unused fields automatically and drop the fgs_program eck table. This will fix the install errors we experienced on Live.

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.install [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * Uninstall fields, tables, and any associated content types
+ */
+function ucgy_fgs_uninstall() {
+  $module = ucgy_fgs;
```
Comment (balrajb) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.install]: it is not an integer, should be in quotes.

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.install [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      }
+      // Check if tables exist and delete if so
+      if(db_table_exists('field_data_'. $field_name)) {
+        db_drop_table('field_data_'. $field_name);
+        db_drop_table('field_revision_'. $field_name);
+      }   
+
+   // Drop FGS Graduate Program Table as part of uninstall
+   db_drop_table('eck_fgs_graduate_program');
+
```
Comment (balrajb) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.install]: extra line can be deleted.

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.install [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      if(field_info_field($field_name)) {
+        field_delete_field($field_name);
+      }
+      // Check if tables exist and delete if so
+      if(db_table_exists('field_data_'. $field_name)) {
+        db_drop_table('field_data_'. $field_name);
+        db_drop_table('field_revision_'. $field_name);
+      }   
+
+   // Drop FGS Graduate Program Table as part of uninstall
```
Comment (balrajb) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.install]: indentation is not correct for these two line, not a big deal though!

**Comment by balrajb** [comment]
thanks for changes @dpfitzsi 

Quick question: when I install ucgy_fgs on my local, getting error table 'eck_fgs_graduate_program' doesn't exist
is this the one we trying to fix in this PR or separate issue.

All said it could be just my local. Let me know if I should do more investigation into this.

**Pull request merged**